packages/esm-service-queues-app/src/hooks/useQueueEntries.ts

```diff
@@ -137,10 +166,8 @@ export function useQueueEntries(searchCriteria?: QueueEntrySearchCriteria, rep:
   }, [pageError]);
 
   const queueUpdateListener = useCallback(() => {
-    setWaitingForMutate(true);
-    setCurrentPage(0);
-    setPageUrl(getInitialUrl(rep, searchCriteria));
-  }, [rep, searchCriteria]);
+    refetchAllData();
+  }, [refetchAllData]);
```

Comment:
This function is not needed now; now you can just use `refetchAllData` directly as the event listener callback.

Reply:
Hi, @brandones!
I have this wrapping function because the event listener has the signature of (event) => void, and `refetchAllData` has the signature of `searchCriteria` and `rep`.
Thanks!